packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts

```diff
@@ -28,11 +29,30 @@ export class W3mConnectingWcView extends LitElement {
 
   @state() private platforms: Platform[] = []
 
+  @state() private isSiweEnabled = OptionsController.state.isSiweEnabled
+
+  private unsubscribe: (() => void)[] = []
+
   public constructor() {
     super()
     this.determinePlatforms()
     this.initializeConnection()
     this.interval = setInterval(this.initializeConnection.bind(this), ConstantsUtil.TEN_SEC_MS)
+    this.unsubscribe.push(
+      AccountController.subscribe(val => {
+        if (val.siweStatus === 'authenticating') {
+          SnackController.showLoading('Authenticating', 8000)
+        }
+
+        if (val.siweStatus === 'success') {
+          SnackController.hide()
+        }
+        if (val.siweStatus === 'ready') {
+          SnackController.hide()
+        }
```

Comment:
what about other states? do they need handling?

Reply:
Other states don't need to be handled.